body.tex

```diff
@@ -48,80 +50,133 @@ \section{Rubin Operations Leadership and Annual Reporting}
+% \caption{The level 2 and level 3 elements of the Rubin Operations work breakdown structure.}
+% \label{tab:wbs}
+% \end{table}
 
 %\subsection{The Control Account Manager}
 %\label{sec:cam}
 
-Detailed work is planned in advance of each financial year at the team level.
+Detailed work is planned in advance of each fiscal year at the team level.
 Team leads will work with department associate directors to develop plans for activities that address specific milestones, projects, and level of effort activity.
-The highest level milestones are reported regularly throughout the fiscal year to SLAC, AURA, NSF and DOE.
+Progress towards the highest level milestones is reported regularly throughout the fiscal year to SLAC, AURA, NSF and DOE.
 
 
-\subsection{Level of Effort Work}
+\subsection{Activity Types}
 \label{sec:loe}
 
-There is work throughout Rubin Operations that is recorded as a \gls{LOE}.
-These activities include attending meetings, reporting on milestones, or taking part in other activities which do not directly map to deployed code.
-This may be particularly the case for technical managers or others in leadership roles.
-\gls{LOE} work is assumed to earn value (informally) simply through the passage of time.
+There are two types of activities (or epics) that are planned:  activities that result in a deliverable, and level of effort (or support) activities.
+Progress can be tracked on activities with a deliverable by computing the fraction of the effort that is complete (percent complete) based on completed stories linked to the epic.
+Progress on \gls{LOE} work is assumed to progress proportionally with the passage of time.
 
+\gls{LOE} activities include attending meetings, reporting on milestones, or taking part in other activities which do not directly map to a deliverable or product.
+This may be particularly the case for technical managers or others in leadership roles.
 In general, we strive to minimize the fraction of effort which is devoted to \gls{LOE} activities and favor those which are more directly accountable.
 However, in certain cases such as operations and maintenance of telescope and facility systems, pipelines or other systems, \gls{LOE} is perfectly acceptable.
-
-As an example, a first-order estimate is that developers will spent 30\% of their time on \gls{LOE} type activities, and the remaining 70\% of their effort is tracked against concrete deliverables.
-Technical staff in Chile at the summit facility may spend a much more significant fraction of time as  \gls{LOE}.
-However, as above, we generally aspire to minimize the fraction of \gls{LOE} for development activity.
+Technical staff in Chile at the summit facility may spend a much more significant fraction of time as \gls{LOE}.
+As an example, a first-order estimate is that developers will spent 30\% of their time on \gls{LOE} type activities, and the remaining 70\% of their effort is planned and tracked against well-defined deliverables.
```

Comment:
That seems very optimistic for operations where most work is emergent - fix this major issue. Its not planned. Who came up with this 70/30 ?   That is more like construction.

Reply:
Yes, the 70/30 is a carry over assumption from Construction. What do you suggest we include here for DP and SF folks?

Reply:
50 50 but we can update later